sdk/core/azure-core/inc/azure/core/base64.hpp

```diff
@@ -18,7 +18,7 @@ namespace Azure { namespace Core {
    *@brief Used to convert one form of data  into another, for example encoding binary data into
    *base 64 text.
    */
-  class Convert {
+  class Convert final {
```

Comment:
All these changes are breaking changes and need a changelog entry. @antkmsft please make sure to add one.